--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,6 @@
         'numpy >= 1.7.0',
         'mido >= 1.1.16',
         'six',
-        'importlib_resources',
     ],
     extras_require={
         'fluidsynth': ['pyfluidsynth>=1.3.1',],
--- a/pretty_midi/fluidsynth.py
+++ b/pretty_midi/fluidsynth.py
@@ -3,7 +3,7 @@
 """
 
 import os
-import importlib_resources
+import importlib.resources as importlib_resources
 
 try:
     import fluidsynth
