--- a/tests/test_data.py
+++ b/tests/test_data.py
@@ -17,10 +17,8 @@
 
-from __future__ import unicode_literals  # Python 2 compatibility
-
 import pytest
 
 import parselmouth
 
-from future.utils import text_to_native_str  # Python 2 compatibility
+text_to_native_str = str
 

--- a/tests/test_matrix.py
+++ b/tests/test_matrix.py
@@ -17,6 +17,4 @@
 
-from __future__ import division  # Python 2 compatibility
-
 import pytest
 
 import parselmouth
